KarhooSDK/Api/SDKConfiguration/KarhooSDKConfiguration.swift

```diff
@@ -15,11 +15,18 @@ public protocol KarhooSDKConfiguration {
     func authenticationMethod() -> AuthenticationMethod
 
     func analyticsProvider() -> AnalyticsProvider
+
+    /// Closure to provide new auth credentials for KarhooSDK usage, when the current one expires and there is no refresh token available.
+    func requestNewAuthenticationCredentials(callback: @escaping (Credentials?) -> Void)
 }
 
 public extension KarhooSDKConfiguration {
 
     func analyticsProvider() -> AnalyticsProvider {
         return DefaultAnalyticsProvider()
     }
+
+    func requestNewAuthenticationCredentials(callback: @escaping (Credentials?) -> Void) {
```

Comment:
@AleksanderWedrychowskiKarhoo  The result of the DP login API call is of type UserInfo, not Credentials. Internally, we do get and store them, but we are not returning or exposing the credentials at any point. And we shouldn't. Thinking about having no Credentials parameter here and when they trigger the callback upon login return, we retrieve the credentials ourselves inside the network sdk.
cc: @mateid-kr